framework/src/main/java/org/checkerframework/framework/flow/CFAbstractStore.java

```diff
@@ -940,41 +940,47 @@ public int hashCode() {
     @SideEffectFree
     @Override
     public String toString() {
-        return "Use a CFGVisualizer to see the Store: " + this.hashCode();
+        return "Use visualize(new StringCFGVisualizer<> viz) to see the Store: " + this.hashCode();
```

Comment:
You could simply do this :-) That is, see whether the following would work:
`return visualize(new StringCFGVisualizer<>());`